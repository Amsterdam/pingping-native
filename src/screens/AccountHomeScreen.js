import React from 'react';
import {StyleSheet, FlatList} from 'react-native';
import PropTypes from 'prop-types';
import {Container} from 'native-base';
import SimpleHeader from '../components/header/SimpleHeader';
import ExportImage from '../assets/export.png';
import WarningImage from '../assets/warning.png';
import TrashImage from '../assets/trashcan.png';
import AccountBlockButton from '../components/account/AccountBlockButton';
import {appColors} from '../config/colors';
import {testIDs} from '../../e2e/modulesTestIDs';

const AccountHomeScreen = ({navigation}) => {
  const buttons = [
    {
      title: 'Exporteer gegevens',
      image: ExportImage,
      route: 'ExportData',
      testID: testIDs.ACCOUNT.EXPORT_DATA_BUTTON,
    },
    {
      title: 'Privacy',
      image: WarningImage,
      route: 'ViewPrivacy',
      testID: testIDs.ACCOUNT.VIEW_PRIVACY_BUTTON,
    },
    {
      title: 'Verwijder gegevens',
      image: TrashImage,
      route: 'DeleteData',
      testID: testIDs.ACCOUNT.DELETE_DATA_BLOCK_BUTTON,
    },
  ];
  return (
    <Container style={styles.container} testID={testIDs.ACCOUNT.SCREEN}>
      <SimpleHeader title="Gegevens" />
      <FlatList
        data={buttons}
        contentContainerStyle={styles.flatList}
        columnWrapperStyle={styles.flatListColumn}
        renderItem={({item}) => (
          <AccountBlockButton button={item} navigation={navigation} />
        )}
        numColumns={2}
        keyExtractor={(item) => item.title}
      />
    </Container>
  );
};

const styles = StyleSheet.create({
  flatListColumn: {
    justifyContent: 'space-between',
    paddingHorizontal: 25,
    paddingVertical: 10,
  },
  container: {
    backgroundColor: appColors.almostNotBlue,
  },
  flatList: {
    backgroundColor: appColors.almostNotBlue,
    marginTop: 12,
  },
});

AccountHomeScreen.propTypes = {
  navigation: PropTypes.object.isRequired,
};

export default AccountHomeScreen;
