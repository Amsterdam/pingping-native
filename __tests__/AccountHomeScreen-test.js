import React from 'react';
import AccountHomeScreen from '../src/screens/AccountHomeScreen';
import renderer from 'react-test-renderer';

jest.useFakeTimers();

test('renders correctly', () => {
  const tree = renderer.create(<AccountHomeScreen />).toJSON();
  expect(tree).toMatchSnapshot();
});
