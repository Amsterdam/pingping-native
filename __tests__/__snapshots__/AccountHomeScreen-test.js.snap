// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#fff",
        "flex": 1,
        "height": 1334,
      },
      Object {
        "backgroundColor": "#f7fbff",
      },
    ]
  }
  testID="ACCOUNT_SCREEN"
>
  <View>
    <RCTSafeAreaView
      emulateUnlessSupported={true}
      style={
        Object {
          "backgroundColor": "#0d2036",
        }
      }
    >
      <View
        noShadow={true}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
              "borderBottomColor": "transparent",
              "borderBottomWidth": 0.5,
              "elevation": 0,
              "flexDirection": "row",
              "height": 64,
              "justifyContent": "center",
              "left": 0,
              "paddingLeft": 6,
              "paddingRight": 10,
              "paddingTop": undefined,
              "right": 0,
              "shadowColor": null,
              "shadowOffset": null,
              "shadowOpacity": null,
              "shadowRadius": null,
              "top": 0,
            },
            Object {
              "alignItems": "center",
              "backgroundColor": "#0d2036",
              "flexDirection": "column",
              "justifyContent": "center",
            },
          ]
        }
        transparent={true}
      >
        <Text
          ellipsizeMode="tail"
          numberOfLines={0}
          style={
            Array [
              Object {
                "color": "#000",
                "fontFamily": "System",
                "fontSize": 16,
              },
              Object {
                "fontFamily": "Heavitas",
                "fontSize": 30,
                "textAlign": "left",
              },
              Array [
                Object {
                  "fontSize": 14,
                },
                Object {
                  "color": "#fff",
                },
                false,
              ],
            ]
          }
        >
          Gegevens
        </Text>
      </View>
    </RCTSafeAreaView>
  </View>
  <RCTScrollView
    contentContainerStyle={
      Object {
        "backgroundColor": "#f7fbff",
        "marginTop": 12,
      }
    }
    data={
      Array [
        Object {
          "image": <Export
            style={
              Object {
                "marginBottom": 20,
              }
            }
          />,
          "route": "ACCOUNT_EXPORT_DATA_SCREEN",
          "testID": "EXPORT_DATA_BUTTON",
          "title": "Exporteer gegevens",
        },
        Object {
          "image": <Lock
            style={
              Object {
                "marginBottom": 20,
              }
            }
          />,
          "route": "ACCOUNT_PRIVACY_POLICY_SCREEN",
          "testID": "VIEW_PRIVACY_BUTTON",
          "title": "Privacy",
        },
        Object {
          "image": <Trashcan
            style={
              Object {
                "marginBottom": 20,
              }
            }
          />,
          "route": "ACCOUNT_DELETE_DATA_SCREEN",
          "testID": "DELETE_DATA_BLOCK_BUTTON",
          "title": "Verwijder gegevens",
        },
      ]
    }
    disableVirtualization={false}
    getItem={[Function]}
    getItemCount={[Function]}
    horizontal={false}
    initialNumToRender={10}
    keyExtractor={[Function]}
    maxToRenderPerBatch={10}
    onContentSizeChange={[Function]}
    onEndReachedThreshold={2}
    onLayout={[Function]}
    onMomentumScrollEnd={[Function]}
    onScroll={[Function]}
    onScrollBeginDrag={[Function]}
    onScrollEndDrag={[Function]}
    removeClippedSubviews={false}
    renderItem={[Function]}
    scrollEventThrottle={50}
    stickyHeaderIndices={Array []}
    updateCellsBatchingPeriod={50}
    viewabilityConfigCallbackPairs={Array []}
    windowSize={21}
  >
    <View>
      <View
        onLayout={[Function]}
        style={null}
      >
        <View
          style={
            Array [
              Object {
                "flexDirection": "row",
              },
              Object {
                "justifyContent": "space-between",
                "paddingHorizontal": 25,
                "paddingVertical": 10,
              },
            ]
          }
        >
          <View
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#fff",
                "borderRadius": 10,
                "elevation": 1,
                "justifyContent": "center",
                "opacity": 1,
                "padding": 20,
                "shadowColor": "#000",
                "shadowOffset": Object {
                  "height": 1,
                  "width": 0,
                },
                "shadowOpacity": 0.18,
                "shadowRadius": 1,
                "width": "45%",
              }
            }
            testID="EXPORT_DATA_BUTTON"
          >
            <RNSVGSvgView
              align="xMidYMid"
              bbHeight={56}
              bbWidth={56}
              fill="none"
              focusable={false}
              height={56}
              meetOrSlice={0}
              minX={0}
              minY={0}
              style={
                Array [
                  Object {
                    "backgroundColor": "transparent",
                    "borderWidth": 0,
                  },
                  Object {
                    "marginBottom": 20,
                  },
                  Object {
                    "flex": 0,
                    "height": 56,
                    "width": 56,
                  },
                ]
              }
              vbHeight={56}
              vbWidth={56}
              width={56}
              xmlns="http://www.w3.org/2000/svg"
            >
              <RNSVGGroup
                fill={null}
                propList={
                  Array [
                    "fill",
                  ]
                }
              >
                <RNSVGGroup
                  clipPath="prefix__clip0"
                >
                  <RNSVGPath
                    d="M10.188 23.475v23.168c0 .854.53 1.618 1.329 1.919l16.596 6.235 16.596-6.235a2.05 2.05 0 001.33-1.92v-22.5"
                    fill={4291160787}
                    propList={
                      Array [
                        "fill",
                      ]
                    }
                  />
                  <RNSVGPath
                    d="M46.038 20.255l-17.925-6-17.925 6 17.925 6.908 17.925-6.908z"
                    fill={4294572537}
                    propList={
                      Array [
                        "fill",
                      ]
                    }
                  />
                  <RNSVGPath
                    d="M28.113 27.163l-3.487 5.248a1.77 1.77 0 01-2.128.666L8.336 27.45a1.77 1.77 0 01-.875-2.54l2.727-4.655 17.925 6.908z"
                    fill={4287743653}
                    propList={
                      Array [
                        "fill",
                      ]
                    }
                  />
                  <RNSVGPath
                    d="M28.113 27.163L31.6 32.41a1.77 1.77 0 002.128.666L47.89 27.45a1.77 1.77 0 00.874-2.54l-2.727-4.655-17.925 6.908z"
                    fill={4287743653}
                    propList={
                      Array [
                        "fill",
                      ]
                    }
                  />
                  <RNSVGPath
                    d="M31.105 20.78V10.047h2.891L28.113.984l-5.883 9.063h2.891V20.78h5.984z"
                    fill={4291160787}
                    propList={
                      Array [
                        "fill",
                      ]
                    }
                  />
                  <RNSVGPath
                    d="M49.709 24.359l-2.727-4.657-.115-.16a1.089 1.089 0 00-.443-.31L32.198 14.47v-3.33h1.798a1.094 1.094 0 00.918-1.689L29.03.39a1.094 1.094 0 00-1.835 0L21.312 9.45a1.094 1.094 0 00.918 1.69h1.798v3.329L9.775 19.243c-.232.094-.414.26-.532.463l-2.725 4.653a2.861 2.861 0 001.413 4.108l1.163.462v17.714c0 1.302.82 2.484 2.038 2.942l16.596 6.236a1.095 1.095 0 00.77 0l16.596-6.236a3.158 3.158 0 002.038-2.942V28.929l1.163-.462a2.862 2.862 0 001.413-4.108zM28.113 2.993l3.87 5.96h-.878c-.604 0-1.094.49-1.094 1.094v9.639h-3.796v-9.64c0-.603-.49-1.093-1.093-1.093h-.878l3.87-5.96zm-2.992 18.88h5.984c.604 0 1.093-.489 1.093-1.093v-4.003l10.61 3.551-14.695 5.663-14.695-5.663 10.61-3.551v4.003c0 .604.49 1.094 1.093 1.094zM8.35 26.027a.67.67 0 01.055-.562l2.257-3.854 15.786 6.083-2.733 4.114a.68.68 0 01-.813.254L8.74 26.434a.667.667 0 01-.39-.408zm36.595 20.617a.961.961 0 01-.62.895l-15.118 5.68V47.81a1.094 1.094 0 00-2.187 0v5.407l-15.118-5.679a.961.961 0 01-.62-.895V29.798l10.813 4.296a2.877 2.877 0 003.442-1.077l1.483-2.231v7.131a1.094 1.094 0 102.187 0v-7.131l1.482 2.231a2.878 2.878 0 003.443 1.077l10.813-4.296v16.845zm2.931-20.617a.667.667 0 01-.39.408L33.325 32.06a.68.68 0 01-.813-.254l-2.733-4.114 15.786-6.082 2.257 3.853a.668.668 0 01.056.562z"
                    fill={4278190080}
                    propList={
                      Array [
                        "fill",
                      ]
                    }
                  />
                  <RNSVGPath
                    d="M28.113 43.937c-.463 0-.887-.303-1.036-.742-.148-.438.002-.94.37-1.22.35-.265.832-.296 1.214-.077a1.1 1.1 0 01.53 1.124 1.105 1.105 0 01-1.078.915z"
                    fill={4278190080}
                    propList={
                      Array [
                        "fill",
                      ]
                    }
                  />
                </RNSVGGroup>
                <RNSVGDefs>
                  <RNSVGClipPath
                    name="prefix__clip0"
                  >
                    <RNSVGPath
                      d="M0 0h56v56H0z"
                      fill={4294967295}
                      propList={
                        Array [
                          "fill",
                        ]
                      }
                    />
                  </RNSVGClipPath>
                </RNSVGDefs>
              </RNSVGGroup>
            </RNSVGSvgView>
            <Text
              ellipsizeMode="tail"
              numberOfLines={0}
              style={
                Array [
                  Object {
                    "color": "#000",
                    "fontFamily": "System",
                    "fontSize": 16,
                  },
                  Object {
                    "fontFamily": "Heavitas",
                    "fontSize": 30,
                    "textAlign": "left",
                  },
                  Object {
                    "fontSize": 14,
                    "textAlign": "center",
                  },
                ]
              }
            >
              Exporteer gegevens
            </Text>
          </View>
          <View
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#fff",
                "borderRadius": 10,
                "elevation": 1,
                "justifyContent": "center",
                "opacity": 1,
                "padding": 20,
                "shadowColor": "#000",
                "shadowOffset": Object {
                  "height": 1,
                  "width": 0,
                },
                "shadowOpacity": 0.18,
                "shadowRadius": 1,
                "width": "45%",
              }
            }
            testID="VIEW_PRIVACY_BUTTON"
          >
            <RNSVGSvgView
              align="xMidYMid"
              bbHeight={56}
              bbWidth={56}
              fill="none"
              focusable={false}
              height={56}
              meetOrSlice={0}
              minX={0}
              minY={0}
              style={
                Array [
                  Object {
                    "backgroundColor": "transparent",
                    "borderWidth": 0,
                  },
                  Object {
                    "marginBottom": 20,
                  },
                  Object {
                    "flex": 0,
                    "height": 56,
                    "width": 56,
                  },
                ]
              }
              vbHeight={56}
              vbWidth={56}
              width={56}
              xmlns="http://www.w3.org/2000/svg"
            >
              <RNSVGGroup
                fill={null}
                propList={
                  Array [
                    "fill",
                  ]
                }
              >
                <RNSVGPath
                  d="M17.22 21.187v-6.17c0-5.941 4.833-10.776 10.775-10.776S38.77 9.076 38.77 15.017v6.17h3.422v-6.17C42.192 7.19 35.822.82 27.995.82c-7.829 0-14.197 6.369-14.197 14.197v6.17h3.421z"
                  fill={4292865018}
                  propList={
                    Array [
                      "fill",
                    ]
                  }
                />
                <RNSVGPath
                  d="M31.222 4.241c-.549 0-1.087.042-1.614.121 5.18.781 9.163 5.261 9.163 10.655v6.17h3.226v-6.17c0-5.942-4.834-10.776-10.775-10.776zM29.608.913A14.27 14.27 0 0027.995.82c-7.828 0-14.197 6.369-14.197 14.197v6.17h3.226v-6.17c0-7.283 5.513-13.3 12.584-14.104z"
                  fill={4289846268}
                  propList={
                    Array [
                      "fill",
                    ]
                  }
                />
                <RNSVGPath
                  d="M43.912 20.858H12.078a3.594 3.594 0 00-3.583 3.584v27.154a3.594 3.594 0 003.583 3.584h31.834a3.594 3.594 0 003.583-3.584V24.442a3.594 3.594 0 00-3.583-3.584z"
                  fill={4293378452}
                  propList={
                    Array [
                      "fill",
                    ]
                  }
                />
                <RNSVGPath
                  d="M11.722 51.596V24.442a3.593 3.593 0 013.583-3.584h-3.227a3.594 3.594 0 00-3.583 3.584v27.154a3.593 3.593 0 003.583 3.584h3.227a3.593 3.593 0 01-3.583-3.584z"
                  fill={4291338869}
                  propList={
                    Array [
                      "fill",
                    ]
                  }
                />
                <RNSVGPath
                  d="M8.495 27.625h39v20.788h-39V27.625z"
                  fill={4294301361}
                  propList={
                    Array [
                      "fill",
                    ]
                  }
                />
                <RNSVGPath
                  d="M8.495 27.625h3.227v20.788H8.495V27.625zM27.68 31.501a4.385 4.385 0 00-4.046 4.077 4.367 4.367 0 002.157 4.057c.24.14.383.399.383.676v3.113c0 .75.608 1.357 1.357 1.357h.929c.749 0 1.357-.608 1.357-1.357v-3.113c0-.28.148-.539.39-.68a4.37 4.37 0 00-2.525-8.13z"
                  fill={4293378452}
                  propList={
                    Array [
                      "fill",
                    ]
                  }
                />
                <RNSVGPath
                  d="M29.4 43.424V40.31a.781.781 0 00-.382-.676 4.367 4.367 0 01-2.158-4.057c.108-1.71 1.22-3.167 2.75-3.779a4.373 4.373 0 00-1.93-.298 4.385 4.385 0 00-4.046 4.076 4.367 4.367 0 002.157 4.058c.24.14.383.399.383.676v3.113c0 .75.608 1.357 1.357 1.357h.929c.485 0 .909-.256 1.148-.638a1.344 1.344 0 01-.207-.72z"
                  fill={4291338869}
                  propList={
                    Array [
                      "fill",
                    ]
                  }
                />
                <RNSVGPath
                  d="M27.977 0C19.696 0 12.96 6.737 12.96 15.017v5.021h-.9a4.409 4.409 0 00-4.404 4.404v27.154A4.409 4.409 0 0012.06 56h31.834a4.409 4.409 0 004.403-4.404V40.05a.82.82 0 10-1.64 0v7.543H9.297V28.445h37.36v7.776a.82.82 0 101.64 0v-11.78a4.409 4.409 0 00-4.403-4.403h-.9v-5.021C42.994 6.737 36.257 0 27.977 0zm18.68 49.234v2.363a2.766 2.766 0 01-2.763 2.762H12.06a2.766 2.766 0 01-2.763-2.762v-2.363h37.36zM14.6 15.017C14.6 7.64 20.601 1.64 27.977 1.64c7.375 0 13.376 6 13.376 13.376v5.021h-1.78v-5.021c0-6.394-5.202-11.596-11.596-11.596S16.38 8.623 16.38 15.017v5.021H14.6v-5.021zm29.294 6.662a2.766 2.766 0 012.763 2.763v2.363H9.297v-2.363a2.766 2.766 0 012.763-2.763h14.002a.82.82 0 100-1.64h-8.04v-5.022c0-5.49 4.465-9.955 9.955-9.955s9.955 4.466 9.955 9.955v5.021h-8.041a.82.82 0 000 1.64h14.003z"
                  fill={4278190080}
                  propList={
                    Array [
                      "fill",
                    ]
                  }
                />
                <RNSVGPath
                  d="M27.513 45.602h.928a2.18 2.18 0 002.177-2.178V40.33a5.214 5.214 0 002.55-4.468 5.21 5.21 0 00-1.652-3.797 5.144 5.144 0 00-3.911-1.38 5.197 5.197 0 00-4.809 4.842 5.174 5.174 0 002.54 4.803v3.095c0 1.2.976 2.178 2.177 2.178zm-3.08-9.972a3.552 3.552 0 015.964-2.366 3.516 3.516 0 011.13 2.597 3.567 3.567 0 01-1.755 3.062c-.49.288-.795.82-.795 1.388v3.113c0 .296-.24.537-.536.537h-.928a.537.537 0 01-.537-.537v-3.113c0-.569-.302-1.1-.789-1.384a3.541 3.541 0 01-1.753-3.297z"
                  fill={4278190080}
                  propList={
                    Array [
                      "fill",
                    ]
                  }
                />
              </RNSVGGroup>
            </RNSVGSvgView>
            <Text
              ellipsizeMode="tail"
              numberOfLines={0}
              style={
                Array [
                  Object {
                    "color": "#000",
                    "fontFamily": "System",
                    "fontSize": 16,
                  },
                  Object {
                    "fontFamily": "Heavitas",
                    "fontSize": 30,
                    "textAlign": "left",
                  },
                  Object {
                    "fontSize": 14,
                    "textAlign": "center",
                  },
                ]
              }
            >
              Privacy
            </Text>
          </View>
        </View>
      </View>
      <View
        onLayout={[Function]}
        style={null}
      >
        <View
          style={
            Array [
              Object {
                "flexDirection": "row",
              },
              Object {
                "justifyContent": "space-between",
                "paddingHorizontal": 25,
                "paddingVertical": 10,
              },
            ]
          }
        >
          <View
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#fff",
                "borderRadius": 10,
                "elevation": 1,
                "justifyContent": "center",
                "opacity": 1,
                "padding": 20,
                "shadowColor": "#000",
                "shadowOffset": Object {
                  "height": 1,
                  "width": 0,
                },
                "shadowOpacity": 0.18,
                "shadowRadius": 1,
                "width": "45%",
              }
            }
            testID="DELETE_DATA_BLOCK_BUTTON"
          >
            <RNSVGSvgView
              align="xMidYMid"
              bbHeight={56}
              bbWidth={56}
              fill="none"
              focusable={false}
              height={56}
              meetOrSlice={0}
              minX={0}
              minY={0}
              style={
                Array [
                  Object {
                    "backgroundColor": "transparent",
                    "borderWidth": 0,
                  },
                  Object {
                    "marginBottom": 20,
                  },
                  Object {
                    "flex": 0,
                    "height": 56,
                    "width": 56,
                  },
                ]
              }
              vbHeight={56}
              vbWidth={56}
              width={56}
              xmlns="http://www.w3.org/2000/svg"
            >
              <RNSVGGroup
                fill={null}
                propList={
                  Array [
                    "fill",
                  ]
                }
              >
                <RNSVGPath
                  d="M34.64.82H15.484c-.87 0-1.574.705-1.574 1.574v33.354c0 .869.705 1.573 1.574 1.573h25.018c.868 0 1.573-.704 1.573-1.573V8.255L34.64.82z"
                  fill={4294638335}
                  propList={
                    Array [
                      "fill",
                    ]
                  }
                />
                <RNSVGPath
                  d="M36.133 2.313v33.435c0 .87-.704 1.574-1.573 1.574h5.942c.869 0 1.573-.705 1.573-1.574V8.256l-5.942-5.943z"
                  fill={4293322482}
                  propList={
                    Array [
                      "fill",
                    ]
                  }
                />
                <RNSVGPath
                  d="M34.64 7.504V.82l7.435 7.435h-6.684a.751.751 0 01-.75-.75z"
                  fill={4289389031}
                  propList={
                    Array [
                      "fill",
                    ]
                  }
                />
                <RNSVGPath
                  d="M39.07 55.18H16.916a4.173 4.173 0 01-4.15-3.743L8.964 14.769H47.02l-3.8 36.668a4.173 4.173 0 01-4.15 3.743z"
                  fill={4289705965}
                  propList={
                    Array [
                      "fill",
                    ]
                  }
                />
                <RNSVGPath
                  d="M41.06 14.769l-3.904 37.669a3.057 3.057 0 01-3.04 2.742h4.954a4.172 4.172 0 004.15-3.743l3.8-36.668h-5.96z"
                  fill={4288258520}
                  propList={
                    Array [
                      "fill",
                    ]
                  }
                />
                <RNSVGPath
                  d="M47.877 19.526H8.11a2.366 2.366 0 01-2.367-2.367v-.024A2.367 2.367 0 018.11 14.77h39.768a2.366 2.366 0 012.366 2.366v.024a2.366 2.366 0 01-2.366 2.367z"
                  fill={4289705965}
                  propList={
                    Array [
                      "fill",
                    ]
                  }
                />
                <RNSVGPath
                  d="M47.877 14.769h-3.622v4.757h3.622a2.366 2.366 0 002.366-2.367v-.024a2.366 2.366 0 00-2.366-2.366z"
                  fill={4288258520}
                  propList={
                    Array [
                      "fill",
                    ]
                  }
                />
                <RNSVGPath
                  d="M19.184 48.66a1.477 1.477 0 01-1.471-1.373L16.204 25.92a1.476 1.476 0 112.946-.207l1.508 21.367a1.477 1.477 0 01-1.474 1.581zM25.057 48.66c-.8 0-1.457-.639-1.476-1.442l-.502-21.369a1.476 1.476 0 112.951-.069l.503 21.368a1.477 1.477 0 01-1.441 1.511h-.035zM30.93 48.66l-.036-.001a1.477 1.477 0 01-1.442-1.51l.503-21.37a1.477 1.477 0 012.952.07l-.503 21.368a1.477 1.477 0 01-1.475 1.442zM36.8 48.66a1.477 1.477 0 01-1.474-1.58l1.51-21.37a1.476 1.476 0 012.945.208l-1.51 21.368a1.477 1.477 0 01-1.47 1.373z"
                  fill={4286481847}
                  propList={
                    Array [
                      "fill",
                    ]
                  }
                />
                <RNSVGPath
                  d="M26.87 3.47h-9.122a.82.82 0 100 1.64h9.121a.82.82 0 100-1.64zM26.87 6.72h-9.122a.82.82 0 100 1.64h9.121a.82.82 0 100-1.64zM19.184 49.48a2.283 2.283 0 001.736-.791c.403-.464.6-1.056.557-1.668L19.97 25.652a2.301 2.301 0 00-2.453-2.129 2.282 2.282 0 00-1.573.786 2.281 2.281 0 00-.556 1.667l1.508 21.368a2.303 2.303 0 002.29 2.136zm-2.002-24.096a.65.65 0 01.495-.226c.341 0 .63.265.655.61l1.509 21.368a.652.652 0 01-.607.7.658.658 0 01-.702-.607L17.023 25.86a.65.65 0 01.16-.477zM25.056 49.48l.055-.001a2.282 2.282 0 001.608-.71 2.283 2.283 0 00.634-1.64L26.85 25.76a2.283 2.283 0 00-.716-1.614 2.27 2.27 0 00-1.634-.628 2.28 2.28 0 00-1.608.711 2.28 2.28 0 00-.634 1.64l.503 21.368a2.31 2.31 0 002.295 2.243zm-.977-24.118c.121-.127.284-.2.464-.204h.019a.656.656 0 01.648.64l.503 21.37a.652.652 0 01-.643.671h-.014a.66.66 0 01-.655-.64l-.503-21.37a.65.65 0 01.181-.467zM30.871 49.479h.058a2.31 2.31 0 002.295-2.242l.503-21.369a2.282 2.282 0 00-.634-1.64 2.278 2.278 0 00-1.605-.71h-.058a2.31 2.31 0 00-2.295 2.242l-.503 21.369c-.015.613.21 1.195.634 1.64.424.443.995.695 1.605.71zm-.599-2.312l.503-21.368a.66.66 0 01.655-.64h.016a.65.65 0 01.46.203.65.65 0 01.18.468l-.502 21.368a.66.66 0 01-.655.64h-.016a.652.652 0 01-.64-.67zM36.801 49.48c1.2 0 2.205-.938 2.29-2.136l1.508-21.368a2.3 2.3 0 00-2.13-2.453 2.3 2.3 0 00-2.452 2.13L34.51 47.02A2.3 2.3 0 0036.8 49.48zm-.656-2.343l1.509-21.369a.658.658 0 01.7-.608c.361.026.634.34.609.7L37.454 47.23a.657.657 0 01-1.309-.092zM47.877 13.948H8.11a3.19 3.19 0 00-3.187 3.187v.024a3.19 3.19 0 003.187 3.187h39.768a3.19 3.19 0 003.187-3.187v-.024a3.19 3.19 0 00-3.187-3.187zm1.546 3.211c0 .853-.693 1.547-1.546 1.547H8.11a1.548 1.548 0 01-1.546-1.547v-.024c0-.853.693-1.546 1.546-1.546h39.768c.853 0 1.546.693 1.546 1.546v.024z"
                  fill={4278190080}
                  propList={
                    Array [
                      "fill",
                    ]
                  }
                />
                <RNSVGPath
                  d="M46.272 21.991a.82.82 0 00-.9.731l-2.967 28.63a3.345 3.345 0 01-3.336 3.008H16.916a3.344 3.344 0 01-3.335-3.007l-2.967-28.63a.82.82 0 10-1.632.168l2.967 28.63a4.98 4.98 0 004.967 4.48h22.153a4.981 4.981 0 004.967-4.48l2.967-28.63a.82.82 0 00-.73-.9zM13.91 12.306a.82.82 0 00.82-.82V2.393c0-.415.338-.753.754-.753H33.82v5.863c0 .867.704 1.572 1.57 1.572h5.865v2.408a.82.82 0 101.64 0V8.255a.822.822 0 00-.24-.58L35.22.24a.821.821 0 00-.58-.24H15.484a2.397 2.397 0 00-2.394 2.394v9.091c0 .453.367.82.82.82zM35.46 2.8l4.634 4.634h-4.633V2.8z"
                  fill={4278190080}
                  propList={
                    Array [
                      "fill",
                    ]
                  }
                />
              </RNSVGGroup>
            </RNSVGSvgView>
            <Text
              ellipsizeMode="tail"
              numberOfLines={0}
              style={
                Array [
                  Object {
                    "color": "#000",
                    "fontFamily": "System",
                    "fontSize": 16,
                  },
                  Object {
                    "fontFamily": "Heavitas",
                    "fontSize": 30,
                    "textAlign": "left",
                  },
                  Object {
                    "fontSize": 14,
                    "textAlign": "center",
                  },
                ]
              }
            >
              Verwijder gegevens
            </Text>
          </View>
        </View>
      </View>
    </View>
  </RCTScrollView>
</View>
`;
